actions/building_positioning.py

```diff
@@ -1,4 +1,4 @@
-from sc2.constants import EVOLUTIONCHAMBER
+from sc2.constants import EVOLUTIONCHAMBER, ENGINEERINGBAY
```

Comment:
You put engineering bay for testing probably it can be removed now

Reply:
see other comment